oldlib/prompts.js

```diff
@@ -94,6 +97,24 @@ var that = {
 		return dfd.promise;
 	},
 
+	enterToContinueControLCToExit(message) {
```

Comment:
There's a capital L too many here and in the reference below: enterToContinueControlCToExit